sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer.
+* Add `version.hpp`.
```

Comment:
super nit: If you would like to fix this going forward and for this current changelog, let's use past-tense similar to https://github.com/Azure/azure-sdk-for-cpp/pull/978
https://github.com/Azure/azure-sdk-for-c/releases/tag/1.0.0
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-core_1.0.0-beta.3

The writing style will continue to differ (and we don't really enforce this), so not a big deal either way, but we can try to make an effort on consistent styling. Up to you.

Reply:
Actually, we did have this conversation within our team. It seems that it is just a personal style. If you take a look at [TensorFlow](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) you would find even within one file there are both styles. Search for `Add` will give you both past tense and simple present tense of the changelog.

Reply:
Yes, I understand. Either is fine. Just wanted to mention it, in case we want to try to converge to one style. Leaving it as is, is fine as well :)